### PR TITLE
reference/microcontrollers: update generated machine files

### PR DESCRIPTION
**Pull request by deadprogram**
This PR updates the generated machine files for next release.

### CODE REVIEW

**Comment by deadprogram** [comment]
Any comments from anyone before I merge?

File: content/docs/reference/microcontrollers/nano-rp2040.md [hunk 1]
```diff
@@ -8,13 +8,13 @@ The [Nano RP2040](https://store.arduino.cc/nano-rp2040-connect) is a tiny develo
 ## Interfaces
 
 | Interface | Hardware Supported | TinyGo Support |
-| --------- | ------------- | ----- |Well, I think
```
Comment (aykevl) [inline content/docs/reference/microcontrollers/nano-rp2040.md:11]: Huh, I wonder where that came from. In any case, it's fixed now.

**Comment by aykevl** [comment]
Yes, looks good to me.

File: content/docs/reference/microcontrollers/nano-rp2040.md (hunk 1 above)
Comment (deadprogram) [reply content/docs/reference/microcontrollers/nano-rp2040.md:11]: Oops?

**Comment by deadprogram** [comment]
Thanks! Merging.

**Pull request merged**